pyrevolve/data_analisys/visualize_robot.py

```diff
@@ -47,16 +145,85 @@ async def test_robot_run(robot_file_path: str):
     robot.update_substrate()
     robot.save_file(f'{robot_file_path}.sdf', conf_type='sdf')
 
-    await connection.pause(False)
+    await connection.pause(True)
```

Comment:
Change to False

Reply:
If you test a robot, do you want to immediately start moving or you want the prepare (start recording) and then click play to start the robot?